lib/node_modules/@stdlib/lapack/base/dorg2r/lib/base.js

```diff
@@ -0,0 +1,124 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var dscal = require( '@stdlib/blas/base/dscal' ).ndarray;
+var dlarf1f = require( './dlarf1f.js' );
+
+
+// MAIN //
+
+/**
+* Generates an M-by-N real matrix Q with orthonormal columns. The matrix Q is defined as the first N columns of a product of K elementary reflectors of order M. `Q = H(1) H(2) . . . H(K)` as returned by `dgeqrf`.
+*
+* @private
+* @param {PositiveInteger} M - number of rows in matrix `A`
+* @param {PositiveInteger} N - number of columns in matrix `A`
+* @param {NonNegativeInteger} K - number of elementary reflectors whose product defines the matrix Q
+* @param {Float64Array} A - input matrix
+* @param {integer} strideA1 - stride of the first dimension of `A`
+* @param {integer} strideA2 - stride of the second dimension of `A`
+* @param {NonNegativeInteger} offsetA - index offset for `A`
+* @param {Float64Array} tau - vector of K scalar factors of the elementary reflectors
+* @param {integer} strideTau - stride length for `tau`
+* @param {NonNegativeInteger} offsetTau - index offset for `tau`
+* @param {Float64Array} work - workspace array
+* @param {integer} strideWork - stride length for `work`
+* @param {NonNegativeInteger} offsetWork - index offset for `work`
+* @returns {Float64Array} matrix `A` overwritten with the orthogonal matrix Q
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+* var tau = new Float64Array( [ 0.0, 0.0 ] );
+* var work = new Float64Array( [ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 ] );
+*
+* dorg2r( 3, 2, 2, A, 1, 3, 0, tau, 1, 0, work, 1, 0 );
+* // A => <Float64Array>[ 1.0, 0.0, 0.0, 0.0, 1.0, 0.0 ]
+*/
+function dorg2r( M, N, K, A, strideA1, strideA2, offsetA, tau, strideTau, offsetTau, work, strideWork, offsetWork ) { // eslint-disable-line max-len, max-params
+	var ia1;
+	var ia3;
+	var del;
+	var ia2;
+	var it;
+	var i;
+	var j;
+	var l;
```

Comment:
I would consolidate these loop variables to just `i` and `j` and be consistent with `i` as outer loop variable and `j` as inner loop variable.